sass/variables/_sizing.scss

```diff
@@ -125,3 +125,12 @@ $base-radio-box-shadow: 0 0 0 1px $grey inset, 0 0 0 9px $white inset, 0 0 0 10p
 $hover-radio-box-shadow: 0 0 0 1px $alert-primary-active inset, 0 0 0 9px $white inset, 0 0 0 16px $alert-primary-active inset;
 $checked-radio-box-shadow: 0 0 0 1px $grey inset, 0 0 0 9px $white inset, 0 0 0 16px $alert-primary inset;
 $active-radio-box-shadow: 0 0 0 8px $alert-primary-active inset, 0 0 0 16px $white inset;
+
+// Tables
+$table-margin: $base-margin;
+$table-row-divider-width: 1px;
+$table-header-padding: $base-margin;
```

Comment:
This, and the awkward grey, are both relics gleaned from ecomm's shopping cart (which I don't believe fully follows our branding guidelines). We may not want to ingrain their styling in our style-base, as it may be replaced shortly. Thoughts?